components/raftstore/src/store/local_metrics.rs

```diff
@@ -310,6 +312,12 @@ impl RaftProposeMetrics {
                 .inc_by(self.conf_change as i64);
             self.conf_change = 0;
         }
+        if self.batch > 0 {
+            PEER_PROPOSAL_COUNTER_VEC
```

Comment:
Better to add a metric `PEER_BATCH_PROPOSAL_COUNTER_VEC`.

Reply:
This is also a kind of `proposal`, so I used a existed metric and just add one type for it.

Reply:
OK